File: app/src/main/java/cy/agorise/crystalwallet/viewmodels/validators/UIValidatorListener.java
package cy.agorise.crystalwallet.viewmodels.validators;

import cy.agorise.crystalwallet.viewmodels.validators.validationfields.ValidationField;

/**
 * Created by Henry Varona on 2/10/2017.
 */

public interface UIValidatorListener {

    public void onValidationSucceeded(ValidationField field);
    public void onValidationFailed(ValidationField field);
}

File: app/src/main/java/cy/agorise/crystalwallet/viewmodels/validators/customImpl/interfaces/UIValidatorListener.kt
package cy.agorise.crystalwallet.viewmodels.validators.customImpl.interfaces

import cy.agorise.crystalwallet.viewmodels.validators.customImpl.validationFields.CustomValidationField


/*
* Listener to deliver response to controls from inner validations
* */
interface UIValidatorListener {


    fun onValidationFailed(customValidationField: CustomValidationField)
    fun onValidationSucceeded(customValidationField: CustomValidationField)
}